content/applications/sales/point_of_sale/payment/stripe.rst

```diff
@@ -0,0 +1,94 @@
+  :guilabel:`Method` field with your payment method's name;
+- Set the :guilabel:`Journal` field as :guilabel:`Bank` and the :guilabel:`Use a Payment Terminal`
+  field as :guilabel:`Stripe`;
+- Enter your payment terminal serial number in the :guilabel:`Stripe Serial Number` field;
+- Click on :guilabel:`Don't forget to complete Stripe connect before using this payment method.`
+
+.. image:: stripe/create-method-stripe.png
+
+.. note::
+   - Find your payment terminal serial number under the device or on `Stripe's website <https://dashboard.stripe.com/register>`_.
+
+Connect Stripe to Odoo
+----------------------
+
+Click on :guilabel:`Connect Stripe`. Doing so redirects you automatically to a configuration page.
+Fill in all the information to create your Stripe account and link it with Odoo. Once the forms are
+completed, the :guilabel:`Publishable Key` and the :guilabel:`Secret Key` are automatically loaded
+in Odoo, and your terminal is ready to be used in a POS.
+
+.. image:: stripe/stripe-connect.png
+
+.. important::
+   - When you use **Stripe** exclusively in PoS, you only need the **Secret Key** to use your
+     terminal.
+   - The :guilabel:`State` is inherent to **eCommerce** and can stay set as :guilabel:`Disabled`.
+   - For **On-Premise** users, the :guilabel:`Connect Stripe` button does not work. To retrieve the
+     keys manually, go to `Stripe's website <https://dashboard.stripe.com/register>`_, log in, type
+     `API` in the search bar, and click :guilabel:`Developers > API`.
+
+Configure the payment terminal
+------------------------------
+
+Swipe right on your payment terminal, click :menuselection:`Settings --> enter the admin PIN code
+--> Validate --> Network`, and select your network.
+
+.. note::
+   - The device must be connected to a secured WIFI.
+   - Your **Odoo** database and payment terminal must share the same network.
+   - To access your payment terminal settings, you must enter the admin PIN code. By default, this
+     code is `07139`.
+
+Link the payment method to a POS
+--------------------------------
+
+To add a **payment method** to your point of sale, go to :menuselection:`Configuration -->
+Settings`. Select the POS, go to :menuselection:`Payments --> Payment Methods`, and add your payment
+method for **Stripe**.
+
+Pay with a payment terminal
+===========================
+
+When processing a payment, select :guilabel:`Stripe` as the payment method. Check the amount and
+click on :guilabel:`Send`. Once the payment is successful, the status changes to :guilabel:`Payment
+Successful`. To cancel the payment request, click on :guilabel:`cancel`.
+
+.. note::
+   - | In case of connexion issues between Odoo and the payment terminal, force the payment by
+       clicking on :guilabel:`Force Done`, which allows you to validate the order.
+     | This option is only available after receiving an error message informing you that the
```

Comment:
There are weird gaps between the lines in the note. This option is on a new line but without any spaces between it and the previous line, but the t
![2022-11-15_09-03](https://user-images.githubusercontent.com/104567387/201863370-7cc79719-3400-43b0-9769-3011f598ec07.png)
wo others have a gap between each.